mobile/test/common/integration/BUILD

```diff
@@ -159,12 +160,10 @@ envoy_cc_test_library(
         "@envoy//test/mocks/server:transport_socket_factory_context_mocks",
         "@envoy//test/test_common:environment_lib",
         "@envoy_build_config//:extension_registry",
-    ] + select({
-        "@envoy//bazel:disable_signal_trace": [],
-        "//conditions:default": [
-            "@envoy//source/common/signal:sigaction_lib",
-        ],
-    }),
+    ] + envoy_select_signal_trace(
+        ["@envoy//source/common/signal:sigaction_lib"],
+        "@envoy",
```

Comment:
ditto

Reply:
Indeed it is, for the reasons mentioned above